### PR TITLE
Start using apify-shared for general consts and utils

### LINKED ISSUE #98: Start using apify-shared for general consts and utils
- In https://github.com/apify/apify-shared-python/issues/1 we introduced `apify-shared` library for sharing general constants and utils across our Python projects. Let's start using it.
  - https://github.com/apify/apify-shared-python
  - https://pypi.org/project/apify-shared/

### PR DESCRIPTION
**Pull request by vdusek**
Closes #98

### CODE REVIEW

**Review by jirimoravcik — APPROVED** [review]

**Comment by jirimoravcik** [comment]
There's still some errors from type checks, etc.

I'd suggest releasing new client first and then using it in this PR so you don't have to write type ignores...

**Comment by vdusek** [comment]
> There's still some errors from type checks, etc.
>
> I'd suggest releasing new client first and then using it in this PR so you don't have to write type ignores...

Agreed, thanks.

File: src/apify/consts.py [hunk 1]
```diff
@@ -1,208 +1,5 @@
 import re
 from enum import Enum
```
Comment (fnesveda) [inline src/apify/consts.py:2]: We'll have to do the reexport magic here, same as in client, so it's not a breaking change

**Review by fnesveda — COMMENTED** [review]
Sorry about the late review, I was waiting for the tests to be fixed.

Anyway, just one note - we need to reexport the consts in `apify.consts` so that this is not a breaking change.

And please bump the version and add an entry to `CHANGELOG.md`

**Pull request merged**